pages/subscriptions-revamp.js

```diff
@@ -0,0 +1,225 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { graphql } from '@apollo/react-hoc';
+import { FormattedMessage, injectIntl } from 'react-intl';
+import styled from 'styled-components';
+
+import { generateNotFoundError } from '../lib/errors';
+import { API_V2_CONTEXT, gqlV2 } from '../lib/graphql/helpers';
+
+import AuthenticatedPage from '../components/AuthenticatedPage';
+import Container from '../components/Container';
+import ErrorPage from '../components/ErrorPage';
+import { Flex } from '../components/Grid';
+import I18nFormatters from '../components/I18nFormatters';
+import Loading from '../components/Loading';
+import StyledTag from '../components/StyledTag';
+import SubscriptionsContainer from '../components/subscriptions-v2/SubscriptionsContainer';
+import TemporaryNotification from '../components/TemporaryNotification';
+import { H2, P } from '../components/Text';
+import { withUser } from '../components/UserProvider';
+
+const subsRevampPageQuery = gqlV2/* GraphQL */ `
+  query Subscriptions($collectiveSlug: String) {
+    account(slug: $collectiveSlug) {
+      id
+      slug
+      name
+      type
+      description
+      settings
+      imageUrl
+      twitterHandle
+      orders {
```

Comment:
Should we try to add a filter and only return recurring financial contributions there?

Reply:
Agreed to keep frontend filtering for now, will add API filtering later